test/evaluate/real.cc

```diff
@@ -19,37 +19,30 @@
 #include <cstdlib>
 
 using namespace Fortran::evaluate;
+using namespace Fortran::common;
 
-using Real2 = typename Type<Category::Real, 2>::Value;
-using Real4 = typename Type<Category::Real, 4>::Value;
-using Real8 = typename Type<Category::Real, 8>::Value;
-using Real10 = typename Type<Category::Real, 10>::Value;
-using Real16 = typename Type<Category::Real, 16>::Value;
-using Integer4 = typename Type<Category::Integer, 4>::Value;
-using Integer8 = typename Type<Category::Integer, 8>::Value;
+using Real2 = typename Type<TypeCategory::Real, 2>::Value;
+using Real4 = typename Type<TypeCategory::Real, 4>::Value;
+using Real8 = typename Type<TypeCategory::Real, 8>::Value;
+using Real10 = typename Type<TypeCategory::Real, 10>::Value;
+using Real16 = typename Type<TypeCategory::Real, 16>::Value;
+using Integer4 = typename Type<TypeCategory::Integer, 4>::Value;
+using Integer8 = typename Type<TypeCategory::Integer, 8>::Value;
 
 void dumpTest() {
   struct {
     std::uint64_t raw;
     const char *expected;
-  } table[] = {
-    { 0x7f876543, "NaN 0x7f876543" },
-    { 0x7f800000, "Inf" },
-    { 0xff800000, "-Inf" },
-    { 0x00000000, "0.0" },
-    { 0x80000000, "-0.0" },
-    { 0x3f800000, "0x1.0p0" },
-    { 0xbf800000, "-0x1.0p0" },
-    { 0x40000000, "0x1.0p1" },
-    { 0x3f000000, "0x1.0p-1" },
-    { 0x7f7fffff, "0x1.fffffep127" },
-    { 0x00800000, "0x1.0p-126" },
-    { 0x00400000, "0x0.8p-127" },
-    { 0x00000001, "0x0.000002p-127" },
-    { 0, nullptr }
-  };
+  } table[] = {{0x7f876543, "NaN 0x7f876543"}, {0x7f800000, "Inf"},
+      {0xff800000, "-Inf"}, {0x00000000, "0.0"}, {0x80000000, "-0.0"},
+      {0x3f800000, "0x1.0p0"}, {0xbf800000, "-0x1.0p0"},
+      {0x40000000, "0x1.0p1"}, {0x3f000000, "0x1.0p-1"},
+      {0x7f7fffff, "0x1.fffffep127"}, {0x00800000, "0x1.0p-126"},
+      {0x00400000, "0x0.8p-127"}, {0x00000001, "0x0.000002p-127"},
+      {0, nullptr}};
```

Comment:
Does this project use clang-format?

Reply:
Yes, unfortunately it sometimes makes a mess of the code.

In this case, adding a comma after `{0, nullptr}` would get it back the way it was. I think that would be an improvement.

Reply:
Ok.